notebooks/250-fastcomposer-image-generation/250-fastcomposer-image-generation.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        1. Upload your images. The order of image1 and image2 needs to match the order of the subects in the prompt. You only need 1 image for single subject generation.
</pre><p>do we always should provide 2 subjects? or 1 or more also can be possible?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
It possible to provide 1 or 2 images. For more images the inference interface should be changed.